docs/docs/_getting-started/dev-mode.md

```diff
@@ -16,16 +17,24 @@ Starting geth in dev mode does the following:
 
 ## Start Geth in Dev Mode
 
-You can specify a data directory to maintain state between runs using the `--datadir` option, otherwise, databases are ephemeral and in-memory:
+Note: this command currently does not work as intended in ephemeral mode and must be run with a data dictionary that contains a keystore file with the hard-coded developer account `0x47e172f6cfb6c7d01c1574fa3e2be7cc73269d95`. The easiest way of doing this is to:
```

Comment:
s/dictionary/directory

Reply:
Hmm this is from an older commit, resolving since this line should be gone anyways now I think!

Reply:
Ah yeah, I went through commit for commit. Sorry for the noise.